iocore/eventsystem/unit-tests/test_MIOBufferWriter.cc

```diff
@@ -21,11 +21,21 @@
     limitations under the License.
  */
 
+#if defined(TS_EXCEPTIONS_IN_UT)
```

Comment:
Where does this get set?

Reply:
Nowhere by default.  I'll just take it out

Reply:
Are you going to make this change?